dspace/config/dspace.cfg

```diff
@@ -1442,6 +1445,12 @@ sitemap.engineurls = http://www.google.com/webmasters/sitemaps/ping?sitemap=
 #
 # No known Sitemap 'ping' URL for MSN/Live search
 
+# Define cron for how frequently the sitemap should refresh.
+# Defaults to running daily at 1:15am
+# Remove (comment) out this config to disable the sitemap scheduler
```

Comment:
Another very minor change.  Please update this line to also describe how to disable via local.cfg.  I've also found (via testing) that this config is not refreshed automatically (which isn't a big deal), so we should document that you need to restart Tomcat.

So, let's say something like:
```
# Remove (comment out) this config to disable the sitemap scheduler.
# Sitemap scheduler can also be disabled by setting to "-" (single dash) in local.cfg.
# Keep in mind, changing the schedule requires rebooting your servlet container, e.g. Tomcat.
```